benchmarks/bench.sh

```diff
@@ -440,6 +440,9 @@ compare_benchmarks() {
     fi
 
     echo "Comparing ${BRANCH1} and ${BRANCH2}"
+    python3 -m venv ./${SCRIPT_DIR}/venv
```

Comment:
I think automatically creating / removing a virtual env each run may be surprising to people already managing their enviornment

What would you think about adding a new command that worked like `bench.sh data` like `bench.sh venv`

That might work like

```shell
# creates virtual environment in $SCRIPT_DIR/venv (alongside data)
# prints out a message about how to activate it
$ bench.sh venv
$ source venv/bin/activate.sh # do what bench.sh tells you
...
$ bench.sh run tpch
```

Reply:
I do like that idea a bit more than what I have implemented now. I would be open to taking that route and updating the PR. Thanks for the feedback, its much appreciated!